### PR TITLE
id:6499 comment:updates to Registration page

### PR DESCRIPTION
**Pull request by davidtrussler**
- copy changes
- some conditional code to deal with user coming to the page fresh or from budget planner
- uses new style for newsletter button

### CODE REVIEW

**Comment by Guntrisoft** [comment]
:+1:

**Comment by darokel** [comment]
Could we add a cucumber scenario for this please?

**Comment by darokel** [comment]
:+1: :ship:

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -14,4 +14,8 @@ def authentication_registration_title
     I18n.t(session['authentication_registration_title'],
            default: I18n.t('authentication.registration.title'))
   end
+
+  def authentication_session_set?
```
Comment (asmega) [inline app/helpers/sessions_helper.rb]: is this method name not misleading?

**Pull request merged**